packages/ui/src/components/forms/form/Form.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <b-form novalidate class="position-relative shadow-sm border p-4 bg-white" @submit.prevent="submit">
     <template v-if="showTitle">
-      <h3 data-cy="form-title">
+      <component :is="titleTag" data-cy="form-title" :class="titleClass">
```

Comment:
this only supports html tags though, right? as others would need to get imported somehow

Reply:
ye true you couldnt do a custom component but thinking about this now i think a slot will work better so will update it to use that

Reply:
oh yeah, slot would be best for that